src/Makefile.am

```diff
@@ -0,0 +1,11 @@
+bin_PROGRAMS = snap-run
+snap_run_SOURCES = main.c seccomp_utils.c utils.c
+snap_run_CFLAGS = \
```

Comment:
`make fmt` is indeed gone, I'll bring it back. Make test is the autotools-std `make check` instead.
I see I don't install the udev helper, thanks for spotting that!